web/src/components/schema/Object.vue

```diff
@@ -1,12 +1,37 @@
 <template>
   <div>
     <template v-for="(val, key) in data">
-      <v-row
-        v-if="val || (!val && !omitEmpty)"
-        :key="key"
-      >
-        <strong class="mr-2">{{ objectKey(key) }}:</strong> {{ renderedValue(val) }}
-      </v-row>
+      <template v-if="isObjectArray(val)">
+        <v-row
+          v-if="val || (!val && !omitEmpty)"
+          :key="key"
+          class="mx-1"
+        >
+          <strong class="mr-2">{{ objectKey(key) }}:</strong>
+        </v-row>
+        <template v-for="item in val">
+          <template v-for="(v, k) in item">
```

Comment:
I don't believe you need this `<template>` here; instead you can put the `v-for` directive in the `<v-row>` on line 14.